java/plugins/html-elements-plugin/src/main/java/ru/sbtqa/tag/pagefactory/htmlstepdefs/HTMLStepDefs.java

```diff
@@ -120,7 +114,7 @@ public void findElementInBlock(String block, String elementType, String elementT
         String[] packages = this.getClass().getCanonicalName().split("\\."); 
         String currentLanguage = packages[packages.length - 2]; 
         I18N i18n = I18N.getI18n(this.getClass(), new Locale(currentLanguage));
-        String key = i18n.getKey(elementType);
+        String key = i18n.get(elementType);
```

Comment:
Does it affect anything?

Reply:
I fix it because test was broken, now test pass